Maxis/Controllers/LoginController.cs

```diff
@@ -12,78 +10,51 @@ namespace Maxis.Controllers
 {
     public class LoginController : Controller
     {
-        
         private readonly IUserService _userService;
-
         public LoginController(IUserService userService)
         {
             _userService = userService;
         }
-        public JsonResult Login(string userName, string password)
+
+        public JsonResult Login(LoginViewModel loginModel)
         {
-            //var server = WebConfigurationManager.AppSettings["Ldapserver"];
-            //var ldapUser = WebConfigurationManager.AppSettings["Ldapusername"];
-            //var ldapPassword = WebConfigurationManager.AppSettings["Ldappassword"];
-           
-            try
-            {
-                    var roles = Authentication(new LoginViewModel
-                    {
-                        Password = password,
-                        Username = userName
-                    });
-                return Json(roles, JsonRequestBehavior.AllowGet);
-            }
-            catch (Exception ex)
-            {
-                return Json(ex.Message.ToString(), JsonRequestBehavior.AllowGet);
-            }
-           
+            ValidateUser(loginModel);
```

Comment:
This check is done by LDAP server code.This is not implemented in the code as we dont have acces to that server from our domain.